util/gc.py

```diff
@@ -37,6 +38,14 @@ def get_free():
     stats = os.statvfs(ROOT_DIR)
     return stats.f_bavail * stats.f_bsize / pow(1024,3)
 
+  def onerror(func, path, exc_info):
```

Comment:
Exactly true. There is nothing we can do to remedy this "onerror" from the code.
I will just log the error and move on in this case